toolchain/parser/parser_context.cpp

```diff
@@ -230,37 +231,41 @@ auto Context::SkipPastLikelyEnd(TokenizedBuffer::Token skip_root)
   return std::nullopt;
 }
 
-auto Context::SkipTo(TokenizedBuffer::Token t) -> void {
-  CARBON_CHECK(t >= *position_) << "Tried to skip backwards from " << position_
-                                << " to " << TokenizedBuffer::TokenIterator(t);
-  position_ = TokenizedBuffer::TokenIterator(t);
+auto Context::SkipTo(Lex::TokenizedBuffer::Token t) -> void {
+  CARBON_CHECK(t >= *position_)
+      << "Tried to skip backwards from " << position_ << " to "
+      << Lex::TokenizedBuffer::TokenIterator(t);
+  position_ = Lex::TokenizedBuffer::TokenIterator(t);
   CARBON_CHECK(position_ != end_) << "Skipped past EOF.";
 }
 
 // Determines whether the given token is considered to be the start of an
 // operand according to the rules for infix operator parsing.
-static auto IsAssumedStartOfOperand(TokenKind kind) -> bool {
-  return kind.IsOneOf({TokenKind::OpenParen, TokenKind::Identifier,
-                       TokenKind::IntegerLiteral, TokenKind::RealLiteral,
-                       TokenKind::StringLiteral});
+static auto IsAssumedStartOfOperand(Lex::TokenKind kind) -> bool {
+  return kind.IsOneOf({Lex::TokenKind::OpenParen, Lex::TokenKind::Identifier,
```

Comment:
Have you considered using `using` declarations instead of `Lex::` qualifying everything (not in .h files, of course)? That seems like it would be less noisy (not to mention making the diff more manageable).